src/mpid/ch4/src/ch4_progress.h

```diff
@@ -67,32 +67,37 @@ MPL_STATIC_INLINE_PREFIX void MPIDI_check_progress_made_vci(MPID_Progress_state
     }
 }
 
+#define MPIDI_THREAD_CS_ENTER_VCI_OPTIONAL(vci)		\
+  if (state->flag & MPIDI_PROGRESS_NM_VCI_LOCK)		\
+    do {						\
+      MPID_THREAD_CS_ENTER(VCI, MPIDI_VCI(vci).lock);	\
+    } while (0)
+
+#define MPIDI_THREAD_CS_EXIT_VCI_OPTIONAL(vci)		\
+  if (state->flag & MPIDI_PROGRESS_NM_VCI_LOCK)		\
+    do {						\
+      MPID_THREAD_CS_EXIT(VCI, MPIDI_VCI(vci).lock);	\
+    } while (0)
+
+
 /* define MPIDI_PROGRESS to make the code more readable (to avoid nested '#ifdef's) */
 #ifdef MPIDI_CH4_DIRECT_NETMOD
 #define MPIDI_PROGRESS(vci) \
     do { \
         if (state->flag & MPIDI_PROGRESS_NM) {	      \
-	  if (state->flag & MPIDI_PROGRESS_NM_VCI_LOCK)	{	\
-	    MPID_THREAD_CS_ENTER(VCI, MPIDI_VCI(vci).lock);	\
-	  }							\
+	  MPIDI_THREAD_CS_ENTER_VCI_OPTIONAL(vci);		\
 	  mpi_errno = MPIDI_NM_progress(vci, 0);		\
-	  if (state->flag & MPIDI_PROGRESS_NM_VCI_LOCK) {	\
-	    MPID_THREAD_CS_EXIT(VCI, MPIDI_VCI(vci).lock);	\
-	  }							\
+	  MPIDI_THREAD_CS_EXIT_VCI_OPTIONAL(vci);		\
```

Comment:
Was the indentation like this before?

Reply:
Looks like it got messed up in my patch. Should be fixed now.